### ISSUE TITLE
Installation error "Cannot find module 'broccoli-static-compiler'"

### ISSUE THREAD

**Issue by vsymguysung** [mysterlune/ember-pdfjs#16]
I got the following error while running `ember install ember-pdfjs` and `ember serve`.

```
Installed packages for tooling via npm.
Cannot find module 'broccoli-static-compiler'
Error: Cannot find module 'broccoli-static-compiler'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/guysungkim/Documents/emberJS-Projects/bbook/node_modules/ember-pdfjs/index.js:3:17)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```
Any suggestion would be appreciated.
Thanks !

**Comment by mysterlune** [comment]
hi @vsymguysung. you can get around this by doing:

`npm install broccoli-static-compiler --save-dev`

for the moment. i'm looking into why this is failing to install with `ember install ember-pdfjs`... stay tuned!

**Comment by mysterlune** [comment]
ok, figured it out. 

just published a patch `0.4.1` to npm for you to begin using.

there was a bad dependency on `broccoli-static-compiler`. i also removed direct dependency for `broccoli-asset-rev`, which was not being used in a meaningful way.

please let me know if you have further issues (and also please fork and create a PR if you have a fix at any point!)

thanks for using the addon.
